src/vm_core.rs

```diff
@@ -155,6 +161,15 @@ impl PyVM {
                 py,
             );
 
+            if self.vm.borrow_mut().get_signature_builtin().is_ok() {
+                let ecdsa_builtin = globals
+                    .get_item("ecdsa_builtin")
+                    .unwrap()
+                    .extract::<PySignature>()
```

Comment:
Can we use eccdsa_builtin.borrow() instead of extracting it from globals?
Similar to how enter_scope and exit_scope are used below

Reply:
Done!